### PR TITLE
Fix #1675 - Console User Interface (work by @raygervais, with fix for regression to UI.js)

### PR DESCRIPTION
**Pull request by humphd**
Fixes to https://github.com/mozilla/brackets/pull/752 to deal with what I had to revert in https://github.com/mozilla/brackets/commit/896adeb1668da2986621cac1b24f2c73f3c840f8.

@raygervais please sign-off on this before I try and merge again.

### CODE REVIEW

**Comment by humphd** [comment]
@raygervais let's move [discussion](https://github.com/mozilla/brackets/pull/672#issuecomment-300333618) here.  What are you seeing with respect to `dist/config.json`?  This is working fine for me locally.

**Comment by raygervais** [comment]
When running the local version, the terminal (backend get requests) show it stopped at `dist/config.json`. I had @flukeout look at it earlier in the week, but we saw nothing wrong with it. If it's running on your computer, along with everyone's except mine, then perhaps my machines are the issue?

**Comment by raygervais** [comment]
image

But it'll stay there until you try to reload again. And so forth.

**Comment by humphd** [comment]
I'd say your dev env is busted.  Take a look at the code and make sure you're OK with what I've done, and I'll try to land it again.  Then we can both test it on staging in the next few days.  We're shipping on the 17th, so we can fix small things we find.

**Comment by raygervais** [comment]
I agree, I say we follow through with your idea & I'll test / setup the development environment again.

**Comment by raygervais** [comment]
I agree, I say we follow through with your idea & I'll test / setup the development environment again.

**Comment by humphd** [comment]
@raygervais so you're going to test this, or you want me to land it and we test on staging?  I'm unclear what you mean above.  I want to get this in today or tomorrow.

**Comment by flukeout** [comment]
Testing now.

File: src/extensions/default/bramble/stylesheets/consoleTheme.less [hunk 1]
```diff
@@ -0,0 +1,279 @@
+#editor-console {
+    position: relative;
+}
+
+#editor-console .console {
+    overflow-x: hidden;
+    max-height: inherit;
+    height: calc(100% - 44px) !important;
```
Comment (flukeout) [inline src/extensions/default/bramble/stylesheets/consoleTheme.less]: @humphd Need to escape the calc rule here...

``height: ~"calc(100% - 44px)" !important;``

**Comment by flukeout** [comment]
@humphd just one small change and I think we're good to go.

**Comment by humphd** [comment]
OK, let's try this again and see if it will stick.  @raygervais, please test for us on staging at http://bramble.mofostaging.net/ and file new bugs on anything you notice.

**Comment by humphd** [comment]
First regression found! https://github.com/mozilla/thimble.mozilla.org/issues/2057

**Pull request merged**